website/src/main/tut/docs/rules/Disable.md

```diff
@@ -5,10 +5,22 @@ title: Disable
 
 # Disable
 
-_Since 0.5.0_
+_Since 0.6.0_
```

Comment:
Let's keep this unchanged and create a new "v0.6.0" section at the bottom that describes the new configuration options.

The website goes live as soon as this is merged so we should not immediately replace the v0.5 documentation, that would be confusing for users.

Reply:
Hmm, how can we keep the old version if we generate it automatically?

Reply:
The stable version is currently hardcoded for 0.5.10 https://scalacenter.github.io/scalafix/docs/users/installation had to do that before releasing 0.6.0-M1 #610

Reply:
And how we can describe the new format then? :D

Reply:
Copy paste the current generated markdown file and inline it. You'll need to generate the site first.